Targets/Jetpack/Sources/AppDelegate+Jetpack.swift

```diff
@@ -0,0 +1,38 @@
+import UIKit
+
+class AppDelegate: WordPressAppDelegate {
+    override func makeBuildSettings() -> BuildSettings {
+        let configuration = XcodeBuildConfiguration.current
+        return BuildSettings(
+            itunesAppID: "1565481562",
+            pushNotificationAppID: configuration.pushNotificationAppID
+        )
+    }
+}
+
+/// - warning: This must be  the only place with conditional compilation.
+private enum XcodeBuildConfiguration {
```

Comment:
The idea is that it will be the only place with conditional compilation, and conditional compilation **must** happen only in the app-targets. This way, you won't have to recompile `WordPressGuts` switching schemas. 

I need to figure out a better way to reuse it.